test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -15,17 +13,20 @@
     private Name name;
     
     @Before
-    public void setup() throws IllegalValueException {
+    public void setup() {
         name = new Name("John K Smith");
     }
     
     @Test
-    public void test_isSimilar_true() {
+    public void test_isSimilar_false() {
         assertFalse(name.isSimilar(null));
+    }
+    
+    @Test
+    public void test_isSimilar_true() {
         assertTrue(name.isSimilar(new Name("John K Smith")));
         assertTrue(name.isSimilar(new Name("John K SMITH")));
         assertTrue(name.isSimilar(new Name("John Smith")));
         assertTrue(name.isSimilar(new Name("Smith, John K")));
     }
```

Comment:
You should put all this test case into different method, where each method name follow our naming convention.